src/components/forms/FamilyForm.tsx

```diff
@@ -488,6 +488,26 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
                   />
                 </FormControl>
               )}
+              {loadedFamily && (
+                <FormControl isRequired isReadOnly isDisabled>
```

Comment:
Wondering if we should display the more meaningful `corpus.title` here?

Reply:
... or maybe both?

Reply:
Done 🖖🏻